scripts/run_backend_tests.py

```diff
@@ -212,23 +212,21 @@ def _get_test_target_classes(path):
     base_path = os.path.join(os.getcwd(), test_path or '')
     result = []
     excluded_dirs = [
-        '.git', 'third_party', 'core/tests', 'node_modules', 'venv']
+        '.git', 'third_party', 'node_modules', 'venv',
+        'core/tests/data', 'core/tests/build_sources']
```

Comment:
Do we have any remaining *_test.py file in the codebase in which we don't want to run the backend test? (Maybe fetch all the tracked files using git and use all the *_test.py files? [To avoid such occurrences in future])

Reply:
I don't think so. All the new *_test.py files being run under this PR are added to the shards JSON file (the run_backend_tests.py script enforces this), so you can look at those changes to see if there is anything unexpected.

One clarification: No test files are excluded by these changes that weren't excluded before. This PR removes `core/tests` and adds `core/tests/data` and `core/tests/build_sources` from the exclusion list, so the end result is to start running tests in `core/tests` that are not in `core/tests/data` or `core/tests/build_sources`